src/models/EmojiUsage.ts

```diff
@@ -0,0 +1,15 @@
+import { Model, Schema, model } from "mongoose";
+
+import { IEmojiUsage } from "types/mongodb";
+
+const EmojiUsageSchema = new Schema({
+  guildId: String,
+  name: String,
+  count: {
+    type: Number,
+    default: 1,
+  },
+  lastUsage: Number,
+});
```

Comment:
Why is the default for `count` 1, when above it was `0` within the `addEmoji` function?
Also, why is the lastUsage a Number and not a Date? (timestamp? messageId? idk)